sdks/python/setup.py

```diff
@@ -101,6 +101,7 @@ def get_version():
 
 REQUIRED_TEST_PACKAGES = [
     'pyhamcrest>=1.9,<2.0',
+    'Sphinx>=1.5.2,<2.0'
```

Comment:
I'm not sure how jenkins runs this. We might just want to install it as part of requirements.